web/src/engine/osk/src/keyboard-layout/oskLayerGroup.ts

```diff
@@ -0,0 +1,60 @@
+import { ActiveLayer } from '@keymanapp/keyboard-processor/build/obj/keyboards/activeLayout.js';
+import { LayoutLayer } from '@keymanapp/keyboard-processor/build/obj/keyboards/defaultLayouts.js';
+import Keyboard from '@keymanapp/keyboard-processor/build/obj/keyboards/keyboard.js';
+import type DeviceSpec from '@keymanapp/web-utils/build/obj/deviceSpec.js';
+
+import OSKLayer from './oskLayer.js';
+import VisualKeyboard from '../visualKeyboard.js';
+
+export default class OSKLayerGroup {
+  public readonly element: HTMLDivElement;
+  public readonly layers: {[layerID: string]: OSKLayer} = {};
+
+  public constructor(vkbd: VisualKeyboard, keyboard: Keyboard, formFactor: DeviceSpec.FormFactor) {
+    let layout = keyboard.layout(formFactor);
+
+    const lDiv = this.element = document.createElement('div');
+    const ls=lDiv.style;
+
+    // Set OSK box default style
+    lDiv.className='kmw-key-layer-group';
+
+    // Return empty DIV if no layout defined
+    if(layout == null) {
+      return;
+    }
+
+    // Set default OSK font size (Build 344, KMEW-90)
+    let layoutFS = layout['fontsize'];
+    if(typeof layoutFS == 'undefined' || layoutFS == null || layoutFS == '') {
+      ls.fontSize='1em';
+    } else {
+      ls.fontSize=layout['fontsize'];
+    }
+
+    ls.width = '100%';
+    ls.height = '100%';
```

Comment:
These seem new here?

Reply:
When developing the OSK module's demo-page, there was a case or two where omission of the newly-added styling resulted in a bad layout for the resulting OSK.  I'm not sure exactly what the problem case was (or "cases were") at this point, though.